src/status_im/chat/views/input/send_button.cljs

```diff
@@ -15,6 +15,9 @@
        (animation/timing spin-value {:toValue  to-spin-value
                                      :duration 300})))))
 
+(defn not-sendable? [input-text]
+  (or (string/blank? input-text) (zero? (compare input-text "/"))))
```

Comment:
Can't this be simplified? Why not just checking for string equality with `/` ?

Reply:
@jeluard I don't think it can be simplified, the `string/blank?` check is needed otherwise the send button will be enabled when the input field is empty. :beers: